cypress/e2e/marc/marc-holdings/add-multiple-001s-edit-marc-holdings.cy.js

```diff
@@ -42,39 +42,39 @@ describe('MARC', () => {
       ]).then((createdUserProperties) => {
         testData.createdUserProperties = createdUserProperties;
 
-        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
-          () => {
-            DataImport.uploadFileViaApi(
-              marcFile.marc,
-              marcFile.fileName,
-              marcFile.jobProfileToRun,
-            ).then((response) => {
-              response.forEach((record) => {
-                recordIDs.push(record[marcFile.propertyName].id);
-              });
-            });
+        DataImport.uploadFileViaApi(
```

Comment:
The user should have permissions for importing file. Please create temperariry user with DI permission and import file.

Reply:
Done